diff -ruN ports.orig/GIDs ports/GIDs
--- ports.orig/GIDs	2011-12-27 14:40:16.000000000 +0200
+++ ports/GIDs	2011-12-27 14:41:43.000000000 +0200
@@ -175,6 +175,7 @@
 openxpki:*:777:
 puppet:*:814:
 jenkins:*:818:
+gerrit:*:819:
 openacs:*:820:
 dotlrn:*:821:
 netdisco:*:840:
diff -ruN ports.orig/UIDs ports/UIDs
--- ports.orig/UIDs	2011-12-27 14:40:11.000000000 +0200
+++ ports/UIDs	2011-12-27 14:41:03.000000000 +0200
@@ -181,6 +181,7 @@
 openxpki:*:777:777::0:0:OpenXPKI Owner:/nonexistent:/usr/sbin/nologin
 puppet:*:814:814::0:0:Puppet Daemon:/nonexistent:/usr/sbin/nologin
 jenkins:*:818:818::0:0:Jenkins CI:/usr/local/jenkins:/usr/sbin/nologin
+gerrit:*:819:819::0:0:Gerrit Daemon User:/usr/local/gerrit:/usr/local/bin/bash
 openacs:*:820:820::0:0:OpenACS Daemon User:/nonexistent:/usr/sbin/nologin
 dotlrn:*:821:821::0:0:.LRN Daemon User:/nonexistent:/usr/sbin/nologin
 netdisco:*:840:840::0:0:netdisco daemon:/nonexistent:/usr/sbin/nologin
